tools/test.py

```diff
@@ -747,14 +748,16 @@ def GetBuildRequirements(self, path, context):
         result += test.GetBuildRequirements(rest, context)
     return result
 
-  def ListTests(self, current_path, path, context, arch, mode):
+  def ListTests(self, current_path, path, context,
+                arch, mode, repeat):
```

Comment:
If you change the method prototype, you also need to update it in the `testcfg.py` and `__init__.py` files under `test/`.  It's probably easier and less intrusive to pass it through the context, it's created in Main() after options have been parsed.